checksumdir/__init__.py

```diff
@@ -46,6 +46,9 @@ def dirhash(dirname, hashfunc='md5', excluded_files=None, ignore_hidden=False,
         if ignore_hidden and re.search(r'/\.', root):
             continue
 
+		dirs.sort()
```

Comment:
This indentation looks too deep.  Shouldn't it be under the `for` loop?

Reply:
Yap it should. I don't know how to use github so i've fixed it in repo. Is it OK now?

Reply:
It actually looks like it is still off by 1 space.  Has this code been tested to work as expected?

Reply:
> It actually looks like it is still off by 1 space. Has this code been tested to work as expected?

On my PC it works fine. Also in my fork it's fine. Maybe you look at outdated version or pull request doesn't work properly?

Reply:
We can leave it to @cakepietoast to make a final determination, but here's what I see:

![Screen Shot 2019-07-24 at 2 55 23 PM](https://user-images.githubusercontent.com/746190/61820872-ea546a80-ae23-11e9-9531-380592342984.png)

Reply:
That's fine. Under for loop. As it must be.

Reply:
If you look closer you will see that it is not indented as it should be.  It is one space closer that the others.  That will throw an `IndentationError` in Python.

Reply:
> If you look closer you will see that it is not indented as it should be. It is one space closer that the others. That will throw an `IndentationError` in Python.
That's what i see. But i can repush.
![2019-07-24_22-51-21](https://user-images.githubusercontent.com/15788974/61824018-aaae7280-ae65-11e9-8ee3-3ca4b6749713.png)

Reply:
That looks correct now, thank you!